install/pre-installation.md

```diff
@@ -690,6 +666,40 @@ Follow the [Prepare Site Init](prepare_site_init.md) procedure.
    cd $HOME && /root/bin/set-sqfs-links.sh
    ```
 
+## 4 Import the CSM Tarball
+
+The following steps require [create system configuration](#3-create-system-configuration) to have completed
+successfully.
+
+1. (`pit#`) Upload the CSM tarball's RPMs and container images to the local Nexus instance.
+
+   ```bash
+   /srv/cray/metal-provision/scripts/nexus/setup-nexus.sh -s
+   ```
+
+1. Add the local Zypper repositories for `noos` and the current SLES distribution.
+
+   > ***NOTE*** The `${releasever_major}` and `${releasever_minor}` variables are interpolated by Zypper, the URI
+   > is intentionally wrapped with single-quotes to prevent the shell from interpolating them. Zypper will replace
+   > these variables with the currently running distributions major and minor version numbers.
+
+   ```bash
+   zypper addrepo --no-gpgcheck --refresh http://packages/repository/csm-noos csm-noos
+   zypper addrepo --no-gpgcheck --refresh 'http://packages/repository/csm-sle-${releasever_major}sp${releasever_minor}' 'csm-sle'
+   ```
+
+1. (`pit#`) Install `csm-testing` RPM.
```

Comment:
Should command say "Install csm-testing, canu, craycli and iuf-cli" RPMs?

Reply:
It sure should.

Reply:
Fixed.